src/app/modules/group/pages/current-user/current-user.component.ts

```diff
@@ -9,14 +11,20 @@ const currentUserBreadcrumbCat = $localize`Yourself`;
   templateUrl: './current-user.component.html',
   styleUrls: [ './current-user.component.scss' ],
 })
-export class CurrentUserComponent implements OnDestroy {
+export class CurrentUserComponent implements OnInit, OnDestroy {
+  currentUser$?: Observable<UserProfile>;
 
   constructor(
     private currentContent: CurrentContentService,
+    private currentUserHttpService: CurrentUserHttpService
   ) {
     this.currentContent.current.next(contentInfo({ breadcrumbs: { category: currentUserBreadcrumbCat, path: [], currentPageIdx: -1 } }));
   }
 
+  ngOnInit(): void {
+    this.currentUser$ = this.currentUserHttpService.getProfileInfo();
```

Comment:
As I wrote in the spec "No need to fetch the data, it has already been loaded by UserSessionService."

Reply:
Fixed